sigil.lic

```diff
@@ -0,0 +1,153 @@
+custom_require.call(%w[common common-arcana common-travel drinfomon equipmanager events])
```

Comment:
(Not blocking merging) It would be nice to have a script-specific link to elanthipedia and some text there about what this is, what it does, etc.

Reply:
Thinking about renaming sigil.lic to sigil-harvest.lic to be more clear about what the script does.  If I do that, should I cancel this PR and open a new one?

Reply:
> Thinking about renaming sigil.lic to sigil-harvest.lic to be more clear about what the script does. If I do that, should I cancel this PR and open a new one?

Since these are all new scripts, I think you're fine to rename locally and push up the renamed scripts. Git _should_ detect it's a rename and not a "delete/create". Just make sure the commit that renames the file doesn't include other changes to the file -- so if you have uncommitted changes, commit those first then rename the file and commit that change.